airflow/providers/amazon/aws/secrets/secrets_manager.py

```diff
@@ -83,43 +90,121 @@ def client(self):
         )
         return session.client(service_name="secretsmanager", **self.kwargs)
 
-    def get_conn_uri(self, conn_id: str) -> Optional[str]:
+    def _get_user_and_password(self, secret):
+        for user_denomination in ['user', 'username', 'login']:
+            if user_denomination in secret:
+                user = secret[user_denomination]
+
+        for password_denomination in ['pass', 'password', 'key']:
+            if password_denomination in secret:
+                password = secret[password_denomination]
+
+        return user, password
+
+    def _get_host(self, secret):
+        for host_denomination in ['host', 'remote_host', 'server']:
+            if host_denomination in secret:
+                host = secret[host_denomination]
+
+        return host
+
+    def _get_conn_type(self, secret):
+        for type_word in ['conn_type', 'conn_id', 'connection_type', 'engine']:
+            if type_word in secret:
+                conn_type = secret[type_word]
+                conn_type = 'postgresql' if conn_type == 'redshift' else conn_type
+            else:
+                conn_type = 'connection'
+        return conn_type
+
+    def _get_port_and_database(self, secret):
+        if 'port' in secret:
+            port = secret['port']
+        else:
+            port = 5432
```

Comment:
Let's not hardcode

Reply:
Check https://github.com/apache/airflow/blob/master/airflow/secrets/local_filesystem.py out